### DISCUSSION TITLE
Welcome to Adonis Discussions!

### DISCUSSION THREAD

**Discussion by Sceleratis** [Epix-Incorporated/Adonis#349] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Please join our Discord community if you just want to chat! https://discord.gg/H5RvTP3

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dimenpsyonal** [comment]
nice yes

**Reply by Sceleratis** [reply]
yis

**Comment by hhhh825** [comment]
How do I make the theme with the UICorner and bind a command to a different rank?

**Reply by hhhh825** [reply]
@Sceleratis

**Comment by hhhh825** [comment]
nvm

**Comment by Expertcoderz** [comment]
## ❗️This forum is currently defunct/unused.

Instead, please join our official and reasonably active Discord server [here](https://discord.gg/H5RvTP3) to communicate and discuss ideas, issues, etc.

Have a great day! To whomever might still be reading this.
